pylint/checkers/typecheck.py

```diff
@@ -219,6 +219,10 @@ def _missing_member_hint(owner, attrname, distance_threshold, max_choices):
               ('Used when a function call does not pass a mandatory'
                ' keyword-only argument.'),
               {'minversion': (3, 0)}),
+    'W1125': ('Keyword argument before variable postional aguments list in %s call of %s function',
+              'keyword-arg-before-vararg',
+              'Used when a keyword argument is defined before *args. There would be '
```

Comment:
The second part of the help message seems a bit confusing. Let's rewrite it instead to ``When defining a keyword argument before positional arguments, one can end up in having multiple values passed for the aforementioned parameter in case the method is called with keyword arguments``

Reply:
One suggestion to your comment - `When defining a keyword argument before **variable** positional arguments....`. This may create confusion about mandatory positional arguments.

Reply:
I'm not sure I understand what kind of confusion.

Reply:
Apologies for not being clear. I meant that we should add the word 'variable' before 'positional arguments' to be crystal clear about that we are talking about *args here and not mandatory positional arguments like var1 in the description. I know we can't have keyword argument before mandatory positional argument but just making it very explicit that its *args we are talking about here. My 2 cents.